package.json

```diff
@@ -2,20 +2,22 @@
   "name": "spellit",
   "version": "1.0.0",
   "description": "Number spelling",
-  "main": "cli.js",
+  "main": "converter.js",
   "scripts": {
-    "test": "ava --watch"
+    "test": "ava",
+    "test:watch": "ava --watch"
   },
   "repository": {
     "type": "git",
-    "url": "https://git.blinksoft.pl/wojciechkalebasiak/slownie"
+    "url": "git+https://github.com/blink-software/spellit.git"
```

Comment:
what is `git+https`?

Reply:
It's the name of the protocol used to install this package, inspired by https://github.com/blink-software/csvsync/blob/master/package.json#L11